### PR TITLE
Home work6

### PR DESCRIPTION
**Pull request by DadizhaO**

### CODE REVIEW

File: src/main/java/HM05/FruitMarket.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            if (fruit.getKinds().equals(kind) && fruit.shelfLife(LocalDate.now())) {
+                fruit.setPrice(fruit.getPrice() * (percent / 100));
+                otherFruits.add(fruit);
+            }
+        }
+        otherFruits.forEach(System.out::println);
+        return otherFruits;
+    }
+
+}
```
Comment (mkharkhuta) [inline src/main/java/HM05/FruitMarket.java:62]: why is this file and some files from the package HW05?

File: src/main/java/HM06/MainLambdaTemplateStudents.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        fruit.setVitamins(Arrays.asList(new Vitamins[]{B, C}));
+
+        fruits.add(fruit);
+
+
+        return fruits;
+
+    }
+
+}
```
Comment (mkharkhuta) [inline src/main/java/HM06/MainLambdaTemplateStudents.java:180]: too many empty lines

**Review by mkharkhuta — APPROVED** [review]

File: src/main/java/HM06/MainLambdaTemplateStudents.java (hunk 2 above)
Comment (SerhiiPasko) [reply src/main/java/HM06/MainLambdaTemplateStudents.java:180]: +1
Bad view

**Review by SerhiiPasko — APPROVED** [review]
Good. In the next time you must create separate branch for separate homework

**Pull request merged**